x/staking/client/cli/utils.go

```diff
@@ -1,12 +1,93 @@
 package cli
 
 import (
+	"encoding/json"
 	"errors"
+	"fmt"
+	"os"
 
+	"cosmossdk.io/math"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
+// validator struct to define the fields of the validator
+type validator struct {
+	Amount            sdk.Coin
+	PubKey            cryptotypes.PubKey
+	Moniker           string
+	CommissionRates   types.CommissionRates
+	MinSelfDelegation math.Int
+}
+
+func parseAndValidateValidatorJSON(cdc codec.Codec, path string) (validator, error) {
+	type internalVal struct {
+		Amount              string `json:"amount"`
+		PubKey              string `json:"pubkey"`
```

Comment:
Shouldn't this be an interface? So you don't need to escape the json in the string.